plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/cart-express-payment.js

```diff
@@ -63,6 +63,19 @@ const CartExpressPayment = () => {
 	) {
 		return null;
 	}
+	/*
+	 * Define the elements used for the Express Payments markup.
+	 *
+	 * When multiple express payment options are available, this will use an
+	 * unordered list to display each option.
+	 *
+	 * When only one express payment option is available, this will use a
+	 * non-sematic DIV for both the wrapper and the individual items. This
+	 * it to prevent accessibility issues caused by a list of one (which isn't
```

Comment:
```suggestion
	 * non-semantic DIV for both the wrapper and the individual items. This
	 * is to prevent accessibility issues caused by a list of one (which isn't
```